plugins/css-has-pseudo/README.md

```diff
@@ -143,11 +143,11 @@ or
 
 ```html
 <!-- When using a CDN url you will have to manually update the version number -->
-<script src="https://unpkg.com/css-has-pseudo@3.0.4/dist/browser-global.js"></script>
+<script src="https://unpkg.com/css-has-pseudo@4.0.0/dist/browser-global.js"></script>
 <script>cssHasPseudo(document)</script>
 ```
 
-⚠️ Please use a versioned url, like this : `https://unpkg.com/css-has-pseudo@3.0.4/dist/browser-global.js`
+⚠️ Please use a versioned url, like this : `https://unpkg.com/css-has-pseudo@4.0.0/dist/browser-global.js`
```

Comment:
I was surprised to see this change because I forgot I made this dynamic. Nice to see that it works 🎉